### PR TITLE
Searching for a school part 2

### PR DESCRIPTION
**Pull request by edwardhorsford**
Adds the non-js school search design history.

Also fixes some dead links in the last post.

### CODE REVIEW

File: app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+## Page variations
+
+{{ appFigure({
+  image: {
+    file: "3. no-results.png",
+    alt: "Content shown when there are no results, prompting the user to change their search."
+  }
+}) }}
+
+{{ appFigure({
```
Comment (jesseyuen) [inline app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md]: Is this supposed to be a duplicate image?

File: app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+The design is the same for single results - but we plan to work on this next.
+
+{{ appFigure({
+  image: {
+    file: "4. single-result.png",
+    alt: "Content shown when there are no results, prompting the user to change their search."
+  }
+}) }}
+
+## Truncating results
```
Comment (jesseyuen) [inline app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md]: This markup isn't rendering as a heading

**Review by jesseyuen — CHANGES_REQUESTED** [review]

File: app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md (hunk 1 above)
Comment (edwardhorsford) [reply app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md]: woops!

File: app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md (hunk 2 above)
Comment (edwardhorsford) [reply app/posts/register-trainee-teachers/2021-05-07-searching-for-a-school-part-2.md]: Super weird! I copied a heading from above (that looks identical) and it should now be fixed.

**Review by jesseyuen — APPROVED** [review]

**Pull request merged**